contracts/core/funds-distribution-token/v1/interfaces/IBasicFDT.sol

```diff
@@ -5,16 +5,44 @@ import "lib/openzeppelin-contracts/contracts/token/ERC20/IERC20.sol";
 
 import "./IBaseFDT.sol";
 
+/// @title BasicFDT implements the basic level FDT functionality for accounting for revenues.
 interface IBasicFDT is IBaseFDT, IERC20 {
 
-    event PointsPerShareUpdated(uint256);
-
-    event PointsCorrectionUpdated(address indexed, int256);
-
-    function withdrawnFundsOf(address) external view returns (uint256);
-
-    function accumulativeFundsOf(address) external view returns (uint256);
-
+    /**
+        @dev   This event emits when the internal `pointsPerShare` is updated.
+        @param pointsPerShare The new value of the internal `pointsPerShare`.
+     */
+    event PointsPerShareUpdated(uint256 pointsPerShare);
+
+    /**
+        @dev   This event emits when an account's `pointsCorrection` is updated.
+        @param account          The address of some account.
+        @param pointsCorrection The new value of the account's `pointsCorrection`.
+     */
+    event PointsCorrectionUpdated(address indexed account, int256 pointsCorrection);
```

Comment:
@deluca-mike did you change any names of event params in the definitions? I saw that some of the function param names changed

Reply:
If so, we will have to communicate all changes to the offchain team

Reply:
Will make sure none change.